Sources/UI/Channels/ChannelsViewController.swift

```diff
@@ -37,10 +37,7 @@ open class ChannelsViewController: ViewController {
     open var presenter = ChannelsPresenter(filter: .currentUserInMembers) {
         didSet {
             reset()
-            
-            if isVisible {
-                setupChannelsPresenter()
-            }
+            setupChannelsPresenter()
```

Comment:
Shouldn't we check `viewDidLoad` was called, since in there this is called again?

Reply:
Thanks! Fixed.